addons/medical/functions/fnc_treatment.sqf

```diff
@@ -163,7 +163,12 @@ if (currentWeapon _caller == secondaryWeapon _caller) then {
     _caller selectWeapon (primaryWeapon _caller);
 };
 
-private _wpn = ["non", "rfl", "pst"] select (1 + ([primaryWeapon _caller, handgunWeapon _caller] find (currentWeapon _caller)));
+private _wpn = switch (true) do {
```

Comment:
it's arma; the commands might return different strings in a different case :smile: